app/assets/javascripts/oxalis/model/sagas/skeletontracing_saga.js

```diff
@@ -182,20 +182,20 @@ export function* diffTrees(
     _.map(trees, tree => tree.treeId),
   );
   for (const treeId of deletedTreeIds) {
-    const prevTree = prevTrees[treeId];
+    const prevTree: TreeType = prevTrees[treeId];
```

Comment:
Doesn't flow find out that `prevTree` is of type TreeType? Seems to me like that should be really easy as flow knows that  `prevTrees` is of type TreeMapType. Or is flow getting confused because of the `_.map` before?

Reply:
Yeah, flow hat problems figuring this out due to the `_.map`. However, I changed the lodash's libdef for `_.map` now. Maybe I can even merge it upstream into the official flowtypes repository. For our code, this should already be quite helpful, though.